### PR TITLE
Fixes #490 Added unit test for twitter.py

### LINKED ISSUE #490: Add unit tests for twitter.py
Add the missing unit tests for twitter.py

### PR DESCRIPTION
**Pull request by dilraj45**
Fixes #490 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: test/test_twitter.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    expected_resp = [
+        {
+            'text': u'mock_text',
+            'link': u'mock_link'
+        }
+    ]
+    mock_requests_get.return_value = mock_response
+    mock_response.json = MagicMock(return_value=dummy_json)
+    resp = Twitter().search('dummy_query', 1)
+    assert expected_resp == resp
```
Comment (vaibhavsingh97) [inline test/test_twitter.py]: Add Blank space in EOF

Comment (realslimshanky) [reply test/test_twitter.py]: I think the file has an EOF otherwise GitHub shows an error icon at the end of the last line.

Comment (dilraj45) [reply test/test_twitter.py]: Yes, it is already there :smile:

Comment (ParthS007) [reply test/test_twitter.py]: @vaibhavsingh97  I agree with @realslimshanky  and @dilraj45 , gihub shows if there is no blank space in EOF

Comment (vaibhavsingh97) [reply test/test_twitter.py]: Sorry! My mistake

**Pull request merged**